### ISSUE TITLE
Track training and validation losses

### ISSUE THREAD

**Issue by Lucky-hqj** [cliport/cliport#13]
hi, authors. Can I track training and validation losses? What should I do if I can?

**Comment by MohitShridhar** [comment]
@Lucky-hqj, if you enable logging [here](https://github.com/cliport/cliport/blob/master/cliport/cfg/train.yaml#L40), you should be able to see train and val losses on wandb. 

In the code, train loss is [here](https://github.com/cliport/cliport/blob/master/cliport/agents/transporter.py#L193) and val loss is [here](https://github.com/cliport/cliport/blob/master/cliport/agents/transporter.py#L272).
